doc/api/stream.md

```diff
@@ -2886,8 +2886,38 @@ readable.on('data', (chunk) => {
 
 #### Piping to writable streams from async iterators
 
-In the scenario of writing to a writable stream from an async iterator, ensure
-the correct handling of backpressure and errors.
+In the scenario of writing to a writable stream from an async iterator, the
+correct handling of backpressure and errors must be ensured.
```

Comment:
A case can be made here for the switch to passive voice but maybe consider keeping it as active voice?:

```suggestion
When writing to a writable stream from an async iterator, ensure correct
handling of backpressure and errors.
```

Reply:
Sure, that works for me! I have changed this.